Userland/Libraries/LibGfx/Rect.h

```diff
@@ -657,7 +657,7 @@ class Rect {
         };
 
         check_distance({ top_left(), top_right() });
-        check_distance({ bottom_left(), bottom_right() });
+        check_distance({ bottom_left().moved_up(1), bottom_right().translated(-1) });
```

Comment:
For consistency, maybe use `.moved_left()` here:
```suggestion
        check_distance({ bottom_left().moved_up(1), bottom_right().moved_left(1) });
```

Reply:
@MacDue that's not the same as translated(-1)

Reply:
Huh, that's a little odd of an API. I feel like we should have `.moved_diagonally()` instead :^)

Reply:
If anything, I'd get rid of moved_left, moved_right, etc. since they are pretty pedantic when you have `.translated(X, Y)`.